templates/core/terraform/deploy.sh

```diff
@@ -18,7 +18,7 @@ LOG_FILE="tmp$$.log"
   -n $TF_VAR_terraform_state_container_name \
   -k ${TRE_ID} \
   -l ${LOG_FILE} \
-  -c "terraform plan -out ${PLAN_FILE} && \
+  -c "terraform plan -out ${PLAN_FILE} && terraform state list"
   terraform apply -input=false -auto-approve ${PLAN_FILE} && \
```

Comment:
Just for my understanding - so apply will now run without the terraform_wrapper script. What does it mean?

Reply:
right - this had been changed for debugging before me, and I didn't realise it was all supposed to feed into the wrapper script - i've reverted now so it's as-was.